packages/server/src/login/login.controller.ts

```diff
@@ -42,6 +43,7 @@ export class LoginController {
       );
       if (!verifySignature) {
         apm.captureError('Invalid request signature');
+        Sentry.captureMessage('Invalid request signature');
```

Comment:
not sure what `parsedRequest` is but should we log it in the message here?

Reply:
sure